metricflow/dataflow/builder/node_evaluator.py

```diff
@@ -116,7 +116,7 @@ def __init__(
         nodes_available_for_joins: Sequence[BaseOutput[SourceDataSetT]],
         node_data_set_resolver: DataflowPlanNodeOutputDataSetResolver[SourceDataSetT],
     ) -> None:
-        """Constructor
+        """Constructor.
```

Comment:
I think the best part of the linter adding this period here is this method is not, in fact, the constructor. 😛

Reply:
lol I need to do a regex replace at some point.